lib/package-card.coffee

```diff
@@ -141,6 +140,12 @@ class PackageCard extends View
         .removeClass('is-disabled')
 
   handlePackageEvents: ->
+    atom.packages.onDidDeactivatePackage (pack) =>
+      @updateEnablement() if pack.name is @pack.name
```

Comment:
`atom.packages.onDidDeactivatePackage` fires for all packages, so this line ensures the package card is only updated if the deactivated package is the same as the package for the card.